hazelcast-openshift-rhel/hazelcast-template.js

```diff
@@ -34,8 +35,16 @@
 					}
 				},
 				"spec": {
+					"volumes":[
+						{
+							"name":"hazelcast-storage",
+							"persistentVolumeClaim":{
+								"claimName":"hz-vc"
+							}
+						}
+					],
```

Comment:
Documentation will be updated, users can input PersistentVolume that are **already created** by OCP itself.